scss/tech.scss

```diff
@@ -0,0 +1,39 @@
+.tech-border {
```

Comment:
You can use `&::before` like we see before in order to render this 'container div' with the green border.
It may look something like this
<img width="406" alt="captura de pantalla 2018-05-31 a la s 14 27 30" src="https://user-images.githubusercontent.com/36163179/40800579-ca933d50-64de-11e8-88da-19d389f0ad90.png">